INSTALL.rst

```diff
@@ -304,13 +304,52 @@ You should also set apache to start on boot::
 
   sudo chkconfig httpd on
 
-The networking server may be started by running::
+Running the network server:
+---------------------------
 
-  haas serve_networks &
 
-as the HaaS user. To make this happen on boot, add the following to ``/etc/rc.local``::
+A systemd script for running the network server is available in the 'scripts' directory.
+Name of the script is: haas_network.service
+
+Centos:
+------
+
+Centos uses systemd to controll all its processes. 
+
+Place the file haas_network.service under:
+``/usr/lib/systemd/system/``
```

Comment:
On my system:

```
# systemctl enable guix-daemon
Created symlink from /etc/systemd/system/mutli-user.target.wants/guix-daemon.service to /etc/systemd/system/guix-daemon.service.
# systemctl disable guix-daemon
Removed symlink /etc/systemd/system/mutli-user.target.wants/guix-daemon.service.
```

Where guix-daemon is a custom `.service` file I've added to `/etc/systemd/system`. Note that the symlink is in the subdirectory `mutli-user.target.wants`. If the service file is provided via an RPM (or deb, in the case of ubuntu), it will go in `/usr/lib/systemd/system` (or perhaps `/lib` on ubuntu), but usually the convention is that scripts not supplied by a package go in `/etc`. I don't know what changes ubuntu has made to the standard convention.